tests/dummy/app/controllers/index.js

```diff
@@ -72,14 +104,40 @@ export default Controller.extend({
   networkFailure : false,
 
   actions : {
-    dragEndAction ({sourceList, sourceIndex, targetList, targetIndex}) {
+    dragEnd ({sourceList, sourceIndex, targetList, targetIndex}) {
       if (sourceList === targetList && sourceIndex === targetIndex) return
 
       const item = sourceList.objectAt(sourceIndex)
 
       sourceList.removeAt(sourceIndex)
       targetList.insertAt(targetIndex, item)
     },
+
+    determineForeignPosition ({draggedItem, items}) {
+      items = A(items.slice()) // create a copy of the list
```

Comment:
This has two issues:

1. `.sortBy` is an Ember method, but there's no guarantee that Ember prototype extensions are enabled in the host app.
2. `.addObject` returns the passed object.